hack/jenkins/job-builder-image/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# This docker image runs Jenkins Job Builder for automatic job reconciliation.
+
+FROM ubuntu:14.04
```

Comment:
Ubuntu has most of the packages we need. I could look into busybox, just haven't used it before.